.clang-format

```diff
@@ -46,13 +46,22 @@ DerivePointerAlignment: false
 DisableFormat:   false
 ExperimentalAutoDetectBinPacking: false
 ForEachMacros:   [ foreach, Q_FOREACH, BOOST_FOREACH ]
+IncludeBlocks: Regroup
 IncludeCategories:
-  - Regex:           '^<'
-    Priority:        3
-  - Regex:           '^"(valhalla)/(baldr|bindings|loki|meili|midgard|mjolnir|odin|proto|sif|skadi|thor|tyr")'
+  - Regex:           '^"'
+    Priority:        1
+  - Regex:           '^<valhalla/'
     Priority:        2
+  - Regex:           '^<(boost|protobuf|sqlite3|curl|zmq)/'
+    Priority:        3
+  - Regex:           '^<.*\.h>'
+    Priority:        3
+  - Regex:           '^<.*\.hpp>'
+    Priority:        3
+  - Regex:           '^<'
+    Priority:        4
   - Regex:           '.*'
-    Priority:        1
+    Priority:        5
 IndentCaseLabels: true
```

Comment:
The order is the following:
- the related header file to the current source (default with priority 0)
- all local includes that are included via `"`
- all valhalla includes that included via `<valhalla/...>`
- all libraries known to me, together with everything that follows `<... .h>` pattern as it looks like a library. For std C includes just use `<c...>` version like `<cstring>`
- std includes that falls into "everything else" category